pkg/tests/api/folders/api_folder_test.go

```diff
@@ -5,32 +5,16 @@ import (
 	"testing"
 
 	"github.com/grafana/grafana-openapi-client-go/models"
-	"github.com/grafana/grafana/pkg/services/featuremgmt"
 	"github.com/grafana/grafana/pkg/tests"
 	"github.com/grafana/grafana/pkg/tests/testinfra"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
 func TestIntegrationUpdateFolder(t *testing.T) {
-	if testing.Short() {
```

Comment:
Same here - these tests were testing both with the feature flag on and with it off. We can consolidate now 🎉